### PR TITLE
Add device set docs

### PR DESCRIPTION
**Pull request by vlindhol**
Merge this when:

- [x] The text is good enough
- [x] The images are using the latest UI changes
- [x] Device Sets are taken out from behind the feature flag

### CODE REVIEW

**Comment by vlindhol** [comment]
Fixed the text and included a screenshot with the latest UI changes. Ready to merge AFAIK!

**Review by rjcase — APPROVED** [review]
Thanks @vlindhol !

File: docs/platform/fleet-wide-metrics.mdx [hunk 1]
```diff
@@ -56,3 +56,74 @@ When comparing Metric Charts, bear in mind that the aggregations `Count` and
...
+2. are running a deprecated software version, or
+3. are reporting a metric over a critical threshold
+
+This kind of followup can easily be done by defining a **Device Set**, which
+will do a daily tally of a search query of your choice, and present it to you in
+a graph:
+
+![](/img/docs/platform/device-set-page.png)
+
+You can see the daily tally for up to 14 days in the past.
```
Comment (HBehrens) [inline docs/platform/fleet-wide-metrics.mdx:76]: Does it make sense to call out (here) in the documentation that the historical data will be built up incrementally?

**Review by HBehrens — APPROVED** [review]
LGTM, one minor nit – we may want to call out in the docs that historical data will not be computed. Effectively, the first data we have access to starts with the creation of the Device Set.

File: docs/platform/fleet-wide-metrics.mdx (hunk 1 above)
Comment (vlindhol) [reply docs/platform/fleet-wide-metrics.mdx:76]: It does. Done!

File: docs/platform/fleet-wide-metrics.mdx [hunk 2]
```diff
@@ -56,3 +56,81 @@ When comparing Metric Charts, bear in mind that the aggregations `Count` and
...
+
+## Device sets
+
+It is often useful to search for devices that fulfill certain criteria and to
+keep track of the number of matches over time. Examples of this could be
+searching for devices that:
+
+1. have not communicated with Memfault within the last N days,
+2. are running a deprecated software version, or
+3. are reporting a metric over a critical threshold
```
Comment (fnune) [inline docs/platform/fleet-wide-metrics.mdx:68]: Ultra nit: I think this last point needs a period at the end.

**Review by fnune — APPROVED** [review]
What I came here to say: https://github.com/memfault/memfault-docs/pull/127/commits/21450fd5e1d7da43679a628687ef474f4148262b

I removed mention of Device Sets from my WIP Metrics doc following Heiko's mentioning that we're not going to get back to Device Sets in a while.

Could you later rebase this PR onto master once I land #127 and kind of revert the commit I linked above?

**Pull request merged**